packages/core/src/amazonqGumby/chat/controller/controller.ts

```diff
@@ -310,7 +310,7 @@ export class GumbyController {
     // prompt user to pick project and specify source JDK version
     private async initiateTransformationOnProject(message: any) {
         const authType = await getAuthType()
-        telemetry.codeTransform_jobIsStartedFromChatPrompt.emit({
+        telemetry.codeTransform_jobStart.emit({
```

Comment:
@sunwoo-ms @dhasani23 this is from https://github.com/aws/aws-toolkit-common/pull/770

Are we ok to start using this in the client?